src/components/Heading/Heading.module.css

```diff
@@ -58,7 +84,8 @@
   color: inherit;
 }
 .heading--neutral {
-  color: var(--eds-theme-color-text-neutral-default);
+  /* TODO: update to text-neutral-default when new colors land */
+  color: var(--eds-theme-color-text-neutral-subtle);
```

Comment:
`text-neutral-default` points to `text-700` which is the darkest color in the current ramp, but not the darkest in the new ramp. meaning in the new ramp, it would be clearly distinct from "black" and "text-800", but currently they're all the same hex value -- so we lighten it for now to keep a visual distinction